app/frontend/src/features/donations/Donations.tsx

```diff
@@ -0,0 +1,123 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import Markdown from "components/Markdown";
+import {
+  DONATIONS_BANNER_TEXT,
+  DONATIONS_BANNER_TITLE,
+  DONATIONS_IMAGE_ALT,
+  DONATIONS_TEXT,
+  DONATIONS_TEXT2,
+  DONATIONS_TITLE,
+} from "features/donations/constants";
+import Landscape from "features/donations/resources/landscape.jpeg";
+import CouchersLogo from "resources/CouchersLogo";
+
+import DonationsBox from "./DonationsBox";
+
+const useStyles = makeStyles((theme) => ({
+  donationsImage: {
+    width: "100%",
+    height: "100%",
+    objectFit: "cover",
+    opacity: "0.3",
+  },
+
+  donationsLogo: {
+    "&&": {
+      height: "72px",
+      width: "initial",
```

Comment:
From memory, I should have used this trick in `Avatar` somewhere related to the `grow` prop.....